src/main/java/com/box/sdk/BoxGroupMembership.java

```diff
@@ -352,6 +381,60 @@ String toJSONString() {
         }
     }
 
+    /**
+     * Enumerates the possible roles that a user can have within a group.
+     */
+    public enum MembershipRole {
```

Comment:
Do you think GroupRole iis a (slightly) better name here?

Reply:
yep, definitely better